iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/checks/DuplicatedEnvironmentVariablesCheck.java

```diff
@@ -23,6 +23,7 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+
```

Comment:
There should be no such line, please check your IDE settings:
https://github.com/SonarSource/sonar-developer-toolset
```
Import Layout
* import all other imports
* <blank line>
* import static all other imports
```

Reply:
Actually that's what I already have, not sure why it added a line here, I will check.
![image](https://github.com/SonarSource/sonar-iac/assets/110470341/dbaca859-6cbe-425a-ae99-42936162238c)